Update live branch with latest changes

dubiety opened the pull request:
* Update WebGL doc
* Remove mobilityURL
* minor changes

herohua committed:
update with existing .openpublishing.publish.config.json

Commit:
CI Update
 Build.Reason:Manual by Taras Vozniuk
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=317140&view=results

ambientlight committed:
initial changelog for azure-maps-control for 2.2.0 (#88)

ambientlight committed:
Merge pull request #87 from MicrosoftDocs/smoke-test

Major Documentation Update: azure-maps-control 2.1.17, azure-maps-indoor 0.1.8, docfx syntax related updates

dubiety committed:
CI Update (#89)

Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=320128&view=results

Co-authored-by: VSC-Service-Account <vscavu@microsoft.com>

dubiety committed:
Merge Smoke test to master - Documentation Update: azure-maps-control 2.2.0 and azure-indoor-maps 0.1.9 (#90)

* CI Update
 Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=320128&view=results

* CI Update
 Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=320218&view=results

Co-authored-by: VSC-Service-Account <vscavu@microsoft.com>

dubiety committed:
Update maps control document (#92)

* CI Update
 Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=320128&view=results

* CI Update
 Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=320218&view=results

* CI Update
 Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=323282&view=results

Co-authored-by: VSC-Service-Account <vscavu@microsoft.com>

opbld32 commented:
Learn Build status updates of commit _[cd87234](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/commits/cd87234356da17b9045e3c9f39c9dc832117d22c)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-cfc2113f)

<a id="cfc2113f"></a>
###
 - **[Error: CannotMergeCommit]** `````Cannot merge commit cd87234356da17b9045e3c9f39c9dc832117d22c in branch main of repository https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript into branch live (commit e680b3bab228912e61f66af433c230afeeeb6dca). Please follow this documentation: https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/ to use git.exe to resolve you content conflicts locally and then push to remote.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8a959d8c-500f-0fdb-1554-1e193b0f705a/202210251038429863-93/BuildReport?accessString=90a0523377083ae280ac27f15dc312b6635d1fca4c043d7eff807679406801f8).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8a959d8c-500f-0fdb-1554-1e193b0f705a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

dubiety committed:
Azure maps typescript docs: Merge latest changes to live  (#91) (#94)

* update with existing .openpublishing.publish.config.json

* CI Update
 Build.Reason:Manual by Taras Vozniuk
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=317140&view=results

* initial changelog for azure-maps-control for 2.2.0 (#88)

* CI Update (#89)

Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=320128&view=results

Co-authored-by: VSC-Service-Account <vscavu@microsoft.com>

* Merge Smoke test to master - Documentation Update: azure-maps-control 2.2.0 and azure-indoor-maps 0.1.9 (#90)

* CI Update
 Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=320128&view=results

* CI Update
 Build.Reason:Manual by Charlie Chen
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=320218&view=results

Co-authored-by: VSC-Service-Account <vscavu@microsoft.com>

Co-authored-by: Ying Hua <herohua@users.noreply.github.com>
Co-authored-by: VSC-Service-Account <vscavu@microsoft.com>
Co-authored-by: Taras Vozniuk <primary.taras.vozniuk@gmail.com>

Co-authored-by: Ying Hua <herohua@users.noreply.github.com>
Co-authored-by: Alma Jenks <v-alje@microsoft.com>
Co-authored-by: Shubha Jain <shubhajain1127@gmail.com>
Co-authored-by: VSC-Service-Account <vscavu@microsoft.com>
Co-authored-by: Taras Vozniuk <primary.taras.vozniuk@gmail.com>
Co-authored-by: Charlie Chen <charlie.chen@microsoft.com>

opbld31 commented:
Learn Build status updates of commit _[c2d3ddf](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/commits/c2d3ddfe449a487cb93abe84e671291474fa9c26)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-dd9af195)

<a id="dd9af195"></a>
###
 - **[Error: CannotMergeCommit]** `````Cannot merge commit c2d3ddfe449a487cb93abe84e671291474fa9c26 in branch main of repository https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript into branch live (commit e680b3bab228912e61f66af433c230afeeeb6dca). Please follow this documentation: https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/ to use git.exe to resolve you content conflicts locally and then push to remote.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8a959d8c-500f-0fdb-1554-1e193b0f705a/202210260302145154-93/BuildReport?accessString=bbd61bce81e9122b302445e8823a841e0f6a0db3c6d7838317bd877f458a4cd8).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8a959d8c-500f-0fdb-1554-1e193b0f705a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

Commit:
Merge branch 'live'

opbld32 commented:
Learn Build status updates of commit _[a0d61c2](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/commits/a0d61c2aaa68c75f01dee3a0f57bfa384e55796f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-autogen/azure-maps-rest/atlas.service.SasCredentialPolicy.yml](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/blob/main/docs-ref-autogen/azure-maps-rest/atlas.service.SasCredentialPolicy.yml) | :warning:Warning | | [Details](#user-content-815808e865d3797aa4966993d515a26a-2da17ec1)
[docs-ref-autogen/azure-maps-rest/atlas.service.MobilityURL.yml](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/blob/main/docs-ref-autogen/azure-maps-rest/atlas.service.MobilityURL.yml) | :white_check_mark:Succeeded |n/a (file deleted or renamed) |
[docs-ref-autogen/azure-maps-rest/atlas.service.SasCredential.yml](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/blob/main/docs-ref-autogen/azure-maps-rest/atlas.service.SasCredential.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-maps-rest/atlas.service.yml](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/blob/main/docs-ref-autogen/azure-maps-rest/atlas.service.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-maps-rest/index.yml](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/blob/main/docs-ref-autogen/azure-maps-rest/index.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/toc.yml](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/blob/main/docs-ref-autogen/toc.yml) | :white_check_mark:Succeeded | |

<a id="815808e865d3797aa4966993d515a26a-2da17ec1"></a>
### [docs-ref-autogen/azure-maps-rest/atlas.service.SasCredentialPolicy.yml](https://github.com/MicrosoftDocs/azure-iot-docs-sdk-typescript/blob/main/docs-ref-autogen/azure-maps-rest/atlas.service.SasCredentialPolicy.yml)
 - **Line 104, Column 51**: **[Warning: xref-not-found]** `````Cross reference not found: 'CredentialPolicy.sendRequest'.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8a959d8c-500f-0fdb-1554-1e193b0f705a/202210260317288302-93/BuildReport?accessString=59535fd93539d1c591a2211f792825af95fbd03f51ec4820731bb82e8b83f703).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8a959d8c-500f-0fdb-1554-1e193b0f705a?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

yulinscottkang reviewed (approved):
💯
